tipg/dbmodel.py

```diff
@@ -181,10 +181,7 @@ def extent(self) -> Optional[Extent]:
                     *[col.bounds for col in cols],
                 ]
 
-            extent["spatial"] = {
-                "bbox": bbox,
-                "crs": f"http://www.opengis.net/def/crs/EPSG/0/{cols[0].srid}",
-            }
+            extent["spatial"] = {"bbox": bbox}
```

Comment:
from https://github.com/developmentseed/tipg/blob/9fbdb31fae359cd780957e6fd1eb3089fb2f20bb/tipg/sql/dbcatalog.sql#L82-L86 it seems that we are always returning bounds in WGS84 (as lon/lat)

cc @bitner